cves/CVE-2017-7668.yml

```diff
@@ -47,8 +47,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
-
+description: |
```

Comment:
Great description! I might change the end where you say "would continue to read past the end of the string in a buffer overrun" to something like "would continue to read past the end of the string, **resulting in a** buffer overrun"

Reply:
Ah, good point, I'll fix this. Thanks!

Reply:
Updated.